src/benchmarks/micro/corefx/System.Xml.Linq/Perf.XElement.cs

```diff
@@ -0,0 +1,56 @@
+﻿using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Xml.Linq
+{
+    [BenchmarkCategory(Categories.CoreFX)]
+    public class Perf_XElement
+    {
+        private XElement _element;
+
+        [Benchmark]
+        public XElement CreateElement() => new XElement("Root", "text node");
+
+        [Benchmark(OperationsPerInvoke = 8)]
+        public XElement CreateWithElements()
+        {
+            XElement doc = new XElement("Root",
+                new XElement("elem1", "text node two e1; text node three"),
```

Comment:
I think the elements should be pre-created in the setup to isolate what you're testing

Reply:
sounds good, please add a comment that this is typical user scenario